### ISSUE TITLE
Show lists expanded by default

### ISSUE THREAD

**Issue by M0E-lnx** [ljmerza/our-groceries-card#19]
Lists are showing up collapsed by default and then automatically collapse when an item is added via the card.

I'd like to see the lists that contain items show up expanded by default and not collapse after an item is added

**Comment by richlee91** [comment]
This would be desirable for me, too or an option where you can define which lists open expanded.   Thank you!!!!

**Comment by esbenlydiksen** [comment]
Thanks for making this component and card - I've been wanting something like this for a long time. I agree that it would be a great improvement to have the possibility to expand lists by default.

**Comment by antonyslater** [comment]
I too would really appreciate the ability to be able to start with a list expanded when the dashboard is loaded, I'm looking to implement this on a non interactive screen for family members to see at a glance what is already on our OurGroceries list.

**Comment by XerxesDGreat** [comment]
@ljmerza I have a fork which addresses this feature request as I also wanted the functionality. Are you open to receiving PRs?

**Comment by ljmerza** [comment]
I don't implement this because the OurGroceries API will be hammered by API requests and since the underlying library isn't official, I dont want to cause issues with OurGroceries.

**Comment by XerxesDGreat** [comment]
Makes sense. Might want to close this issue as "won't do" in that case. I'll continue using my fork then. Thanks!
